propel/schema.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<database name="pizzaService">
+    <table name="ingredients" phpName="Ingredient">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true" />
+    </table>
+
+    <table name="pizzas" phpName="Pizza">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true"/>
+        <column name="price" type="float" required="true"/>
+    </table>
+
+    <table name="pizza_ingredients" phpName="Ingredient_pizza" isCrossRef="true">
+        <column name="ingredients_id" type="integer" primaryKey="true" required="true" />
+        <column name="pizzas_id" type="integer" primaryKey="true" required="true" />
```

Comment:
Foreign column names should always be in singular form, as you only reference **one** object here. So it should be `ingredient_id` and `pizza_id`.